content/spire/try/quickstart.md

```diff
@@ -0,0 +1,209 @@
+---
+title: SPIRE Quickstart
+short: SPIRE Quickstart
+description: How to quickly get SPIRE up and running
+weight: 1
+toc: false
+---
+
+## Overview
+
+In this quick introduction to SPIRE you will learn how to:
+
+* **Start the spire-server process**
+* **Attest a spire-agent to the server using a join token**
+* **Configure a registration entry for a workload**
+* **Fetch an SVID over the WorkloadAPI**
+* **Learn where to find resources for more complex installations on Linux and Kubernetes** 
+
+## Prerequisites
+
+* **A Linux or macOS environment**
+* **The openssl command line tool**
```

Comment:
From where should one obtain this tool if it doesn't come with my distro?

Reply:
It almost certainly is already installed on any Linux or macOS